### PR TITLE
Can now edit the title of a project and also got rid of npm-debug log 

### PR DESCRIPTION
**Pull request by vkr96**
Can now change the title via a text box

### CODE REVIEW

**Comment by agundy** [comment]
Looks good from a preliminary glance. I'll check it out tonight and merge it then if all is well.

**Comment by agundy** [comment]
@vkr96 you also should add the npm debug log to the .gitignore. Looks like this one currently isn't

**Comment by vkr96** [comment]
The previous commit had the npm debug log, I deleted it since I thought it was unnecessary.

**Comment by vkr96** [comment]
Added the npm debug log

**Comment by agundy** [comment]
You misunderstood me! It was good of you to remove the log. I was asking you to add '.npm-debug.log' to the '.gitignore' file so that no one accidentally added the file back to the repository!

**Comment by agundy** [comment]
Just ran this locally. Editing works! I think it would look better if we moved this up to where the actual title of the project is though. see https://github.com/rcos/Observatory3/tree/edit_title

**Comment by agundy** [comment]
screenshot from 2015-09-22 21 55 51
screenshot from 2015-09-22 21 58 51
Sample images.

**Comment by agundy** [comment]
Would be even better if it hid the <h1> name while in edit mode for the title. Should not be hard at all!

File: client/app/projects/profile/profile.controller.js [hunk 1]
```diff
@@ -45,6 +50,17 @@ angular.module('observatory3App')
         });
     };
 
+    $scope.saveName = function(){
+        $scope.edittingName = false;
+        $http.put('/api/projects/' + $scope.project._id, {
+            'name': $scope.project.name
+        }).success(function(){
+            window.alert('Project Name updated!');
```
Comment (seveibar) [inline client/app/projects/profile/profile.controller.js]: we need to eventually change this to notify

**Comment by vkr96** [comment]
Added a donate link, goes to RPI donation page

**Comment by agundy** [comment]
are we closing this and using the newer one?

**Comment by vkr96** [comment]
Yeah the newer one is better.

**Pull request closed without merging**